src/teleop/jetson/jetson_teleop.py

```diff
@@ -110,16 +112,24 @@ def __init__(self):
 
         self.ra_cmd = JointState(
             name=[name for name in RA_NAMES],
-            position=[nan for i in range(len(RA_NAMES))],
-            velocity=[0.0 for i in range(len(RA_NAMES))],
-            effort=[nan for i in range(len(RA_NAMES))],
+            position=[nan for _ in range(len(RA_NAMES))],
+            velocity=[0.0 for _ in range(len(RA_NAMES))],
+            effort=[nan for _ in range(len(RA_NAMES))],
```

Comment:
Honestly if we're changing this, idk why we don't just say 0.0 (or nan) `for _ in RA_NAMES`

Reply:
Applies below as well